src/blocks/box/Box.types.ts

```diff
@@ -0,0 +1,53 @@
+import { CSSProperties, ReactNode } from 'react';
+
+import { ResponsiveProp, SpaceType, ValueOf } from '../Blocks.types';
+import { FlattenSimpleInterpolation } from 'styled-components';
+
+export type BoxResponsiveProps = {
+  alignItems?: ResponsiveProp<CSSProperties['alignItems']>;
+  display?: ResponsiveProp<CSSProperties['display']>;
+  height?: ResponsiveProp<string>;
+  justifyContent?: ResponsiveProp<CSSProperties['justifyContent']>;
+  m?: ResponsiveProp<SpaceType>;
+  maxH?: ResponsiveProp<string>;
+  minH?: ResponsiveProp<string>;
+  maxW?: ResponsiveProp<string>;
+  minW?: ResponsiveProp<string>;
+  p?: ResponsiveProp<SpaceType>;
+  width?: ResponsiveProp<string>;
+};
```

Comment:
- flex direction should also be added here as reponsive prop.

Reply:
Will add it.

Reply:
There should be more properties added like background and border, but we can add them as needed i guess

Reply:
Both are already there

Reply:
https://github.com/push-protocol/push-dapp/pull/1590/files#diff-7d949045d17e37e448062b4981686c38b96e2c3f5177aaae4838755ca622b7bcR20-R28